### ISSUE TITLE
Support for Nuget.config with repositoryPath

### ISSUE THREAD

**Issue by danielcbock** [rsuter/NuGetReferenceSwitcher#14]
If we have a nuget.config file with a repositoryPath for a directory than is not named "packages" the nuget package references are not recognized as nuget package references and you are not presented with the option to change them to project references.

Here is a Nuget.Config that I have for my solution:

<configuration>
  <config>
    <add key="repositorypath" value="..\..\..\NuGetPackages" />
  </config> 
.
.

I see in the ProductModel.cs (LoadReferences method) the Path of the vsReference (e.g. does it contain "\\packages\\" in the path) is used to determine if a reference is from a nuget package.  If a directory other than "packages" (the default) is used then you don't get the option to switch those references to project references.  A simple option would be to let the user specify a regular express or at least a string to use.  Another more complicated option would be to search for a nuget.config and see if it has a repositoryPath.

**Comment by danielcbock** [comment]
I don't think nuget.config appeared in original comment:

&lt;configuration&gt;
  &lt;config&gt
    &lt;add key="repositorypath" value="..\..\..\NuGetPackages" /&gt
  &lt;/config&gt

**Comment by mdvtj** [comment]
I agree with @danielcbock, same situation here (using a shared packages directory for multiple solution via a custom `repositorypath` in a `NuGet.config` in some parent directory).

**Comment by Leon99** [comment]
+1
